frontend/src/types/ontologyTypes.ts

```diff
@@ -5,6 +5,22 @@ export type Annotation = {
   description: string;
 };
 
+export type SustainabilityGoal = {
+  instancesOf: string;
+  label: string;
+  icon: string;
+};
+
+export type SustainabilityGoals = {
+  sustainabilityGoalList: [
+    {
+      instancesOf: string;
+      label: string;
+      icon: string;
+    },
+  ];
+};
```

Comment:
```suggestion
export type SustainabilityGoals = Array<SustainabilityGoal>
```

Unless there is a reason for this type to have the list tied to an attribute the object itself should just be an array of SustainabilityGoals

Reply:
Is the type necessary at all? Can't we just use Array<SustainabilityGoal> instead?

Reply:
Yeah having a separate type is redundant and just wrapping them in an array where they are used is better practice if I'm not mistaken.